auth-api/src/auth_api/schemas/affiliation_invitation.py

```diff
@@ -29,8 +29,9 @@ class Meta(BaseSchema.Meta):  # pylint: disable=too-few-public-methods
         model = AffiliationInvitationModel
         fields = (
             'id', 'from_org', 'to_org', 'business_identifier', 'recipient_email', 'sent_date', 'expires_on',
-            'accepted_date', 'status', 'token', 'type', 'affiliation_id')
+            'accepted_date', 'status', 'token', 'type', 'affiliation_id', 'entity')
 
     from_org = fields.Nested('OrgSchema', only=('id', 'name', 'org_type'))
     to_org = fields.Nested('OrgSchema', only=('id', 'name', 'org_type'))
+    entity = fields.Nested('EntitySchema', only=('business_identifier', 'name'))
```

Comment:
You can't rely on the name here - we're axing this data shortly.

Reply:
Is it going away entirely from the table?
If so, how do I retrieve this data?

Reply:
```
business = AffiliationInvitation._get_business_details(business_identifier,
                                                                   RestService.get_service_account_token())                                                                   ```

Reply:
If you're looking at querying multiple business_identifiers, look at `LEAR_AFFILIATION_DETAILS_URL` and trace it up

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/fb60afce-f933-410c-aa88-64714a2cd3e4)

![image](https://github.com/bcgov/sbc-auth/assets/3484109/38762ff2-7d36-4735-be74-733da046bc8a)

Reply:
Thanks for sharing this !